app/errors.js

```diff
@@ -36,11 +36,21 @@ exports.noAlbum = {
 };
 
 exports.alreadyBought = {
-  statusCode: 404,
+  statusCode: 422,
   message: 'The user has already bought the book before.'
 };
 
 exports.albumsProviderFail = {
-  statusCode: 500,
+  statusCode: 422,
   message: 'The request to the albums provider failed.'
 };
+
+exports.noUserEqual = {
+  statusCode: 422,
+  message: 'The Id of request is not equal to the user authenticated.'
+};
+
+exports.isNotAdmin = {
+  statusCode: 422,
```

Comment:
Same here 401 or 403, because the problema is that the user has not the enough privileges to do that action.